src/os/bluestore/BitmapAllocator.cc

```diff
@@ -36,22 +36,26 @@ int64_t BitmapAllocator::allocate(
   if (!allocated) {
     return -ENOSPC;
   }
-  for (auto i = old_size; i < extents->size(); ++i) {
-    auto& e = (*extents)[i];
-    ldout(cct, 10) << __func__
-                   << " extent: 0x" << std::hex << e.offset << "~" << e.length
-		   << "/" << alloc_unit << "," << max_alloc_size << "," << hint
-		   << std::dec << dendl;
+  if (cct->_conf->subsys.should_gather<dout_subsys, 10>()) {
```

Comment:
The conditional here makes my spidey-sense tingle.  It might be fine, but we should probably make sure this isn't causing overhead vs checking and updating a bool outside allocate.